### ISSUE TITLE
Config so i can point the musicCatalog  to pull off my Google Drive shared folder

### ISSUE THREAD

**Issue by gedw99** [rohansohonee/ufmp#5]
**Describe the bug**

I cant see where i can implement a config to use my google drive shared folder.
I checked the audio plugin and it supports playing from an URL.

But from what i can see in the data model (https://github.com/rohansohonee/ufmp/blob/main/lib/data/musicCatalog.dart) , it wants to get a JSON list of audio files and the meta data about them.
I can easily write a script to loop my google drive files to produce a big JSON file that holds all the Music meta data.

Looking for advice on how to give it a URL to load up the musicCatalog from a certain URL.
I can see you have the URL hardcoded here: https://storage.googleapis.com/uamp/catalog.json

So i think if this can be a config then i can point it to my GDrive.
Maybe also surface it in a Setting Preference too ?

**Comment by rohansohonee** [comment]
@gedw99 
The musicCatalog is created based on the uamp json data. This sample does not cover on how you retrieve your data but only demonstrates on background audio playback using the audio plugins mentioned in the credits section.
I am closing this issue as it is not related to the samples intentions.

**Comment by gedw99** [comment]
Thanks 

I got it working of my google drive, by writing a script to transform my music meta data to match your Schema

I also swapped out the Cache plugin to use the HIVE Cache that does not use SQLite, and is much faster and it woks on Web, Desktop and Mobile.
https://pub.dev/packages/flutter_cache_manager_hive

The Audio Player only works on Mobile. Can get it working on Web later, but no time right now.

I will work and add these if you want. Let me know if you want a PR.
